add new external posts (#278)
cilium/cilium.io: src/posts/2023-05-24-cilium-course/index.md

---
date: '2023-05-24T15:56:00.000Z'
title: 'An inside look at the new Intro to Cilium course with course creator, Jef Spaleta!'
ogImage: ogimage.png

> ogSummary: 'Hear about Cilium directly from Jef Spaleta, the creator of the new Intro to Cilium course'

Reviewer: ```suggestion
ogSummary: 'Hear why Jef Spaleta created the new Intro to Cilium course'
```
